docs/resources/samples/excel-samples.md

```diff
@@ -146,6 +146,68 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
+### Unhide All Rows and Columns
+
+This script get the worksheet's used range, checks if there are any hidden rows and columns, and unhides them. 
+
+```Typescript
+function main(workbook: ExcelScript.Workbook) {
+    // Get the currently selected sheet.
+    const selectedSheet = workbook.getActiveWorksheet();
+
+    // Get the entire data range.
+    const range = selectedSheet.getUsedRange();
+
+    // If the used range is empty, end the script.
+    if (!range) {
+      console.log(`No data on this sheet.`)
+      return;
+    }
+
+    // Log the address of the used range.
+    console.log(`Used range for the worksheet: ${range.getAddress()}`);
+
+    // If no columns are hidden, log message, else, unhide columns
+    if (range.getColumnHidden() == false) {
+      console.log(`No columns hidden`);
+    } else {
+      range.setColumnHidden(false);
+    }
+
+    // If no rows are hidden, log message, else, unhide rows.
+    if (range.getRowHidden() == false) {
+      console.log(`No rows hidden`);
+    } else {
+      range.setRowHidden(false);
+    }
```

Comment:
```suggestion
    // Unhide any hidden rows.
    if (range.getRowHidden()) {
      range.setRowHidden(false);
    } else {
      console.log(`No hidden rows.`);
    }
```

Reply:
Fair enough. In that case, please change `range.getRowHidden() == false` to `!range.getRowHidden()`

Reply:
See above on columns, same issue.